variables.json

```diff
@@ -22,6 +22,11 @@
         "username": "",
         "password": "",
         "secret": ""
+      },
+      "ldap_user": {
```

Comment:
Do we have a test LDAP account for this with heaviliy restricted access? I'm concerned about storing LDAP credentials for test use. Is it possible to create a test user on staging for these tests? Perhaps @johngian has some thoughts on this?

Reply:
hey @davehunt ! I already have an LDAP test account from Infosec, I will use that one (and share it with you). I also have a mozillians account created in staging for the LDAP test account.